18F/identity-idp
# Add BIMI ("Tiny Portable Secure") SVG logo

## 🛠 Summary of changes

Adds a logo within `public/` so that it can be referenced in BIMI DNS records.

References:

- https://bimigroup.org/creating-bimi-svg-logo-files/
- https://docs.aws.amazon.com/ses/latest/dg/send-email-authentication-bimi.html

TBD:

- [ ] DevOps pull request to include path as part of CloudFront cached assets

## 📜 Testing Plan

1. Go to http://localhost:3000/images/login-icon-bimi.svg
2. See a logo
